serving/docker/partition/sm_neo_trt_llm_partition.py

```diff
@@ -156,6 +158,7 @@ def main():
 
         try:
             kwargs["trt_llm_model_repo"] = compiled_model_directory
+            kwargs["cache_root"] = neo_cache_dir
```

Comment:
'cache_root' is generic, and in the future if the cache_root for neo were to behave differently than another cache_root, it would conflict. 

Can we rename to something that is neo-specific?

Reply:
How about `neo_cache_root` ? I am open to suggestions !

Reply:
It seems `NEO_CACHE_DIR` was chosen as an env variable that the service will set. If we could re-use the same i.e. `neo_cache_dir`, that will be good, but if not `neo_cache_root` seems good enough to me